__tests__/articles.test.js

```diff
@@ -51,6 +51,31 @@ describe("/api/articles", () => {
             });
           });
       });
+
+      it("should return articles filtered by topic when topic query is provided", () => {
+        return request(app)
+          .get("/api/articles?topic=coding")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body).toHaveProperty("articles");
+            expect(body.articles).toBeInstanceOf(Array);
```

Comment:
Don't need to check the returned value is an array because we HAVE to return an array for the forEach on line 62 to work, we're doing more asserting than we need